app/views/sensors/show.html.haml

```diff
@@ -8,6 +8,12 @@
 %p
   %strong Address:
   = @sensor.address
+
+- if user_signed_in? && @sensor.animal_id?
```

Comment:
Is the `animal_id` classified or why is it not displayed unless signed in? Just curious.